docker-compose.yml

```diff
@@ -26,7 +26,8 @@ services:
                 python_version: ${PYTHON_VERSION}
         image: elifesciences/digests:${IMAGE_TAG}
         #command: ./migrate-placeholder-script.sh && venv/bin/newrelic-admin run-program venv/bin/uwsgi --ini=/srv/digests/uwsgi.ini --enable-threads
-        command: venv/bin/uwsgi --ini=/srv/digests/uwsgi.ini --enable-threads
+        command: >
+            bash -c "./migrate.sh && venv/bin/uwsgi --ini=/srv/digests/uwsgi.ini --enable-threads"
```

Comment:
can this become the default command of the image? No strong opinions

Reply:
Me neither, I suppose it keeps the image slightly more flexible if this custom command is here, you could be handling the migration(s) elsewhere. I think its fine for now.